DEVELOPERS.md

```diff
@@ -235,16 +235,7 @@ In that situation you can follow the steps below to set up your local copy of th
 1. Give your user the `StaffAreaAdministrator` role by running:
 
    ```sh
-   > python3 manage.py shell_plus
-   ```
-
-   and then running in the `shell_plus` shell:
-
-   ```
-   >>> from jobserver.authorization import StaffAreaAdministrator
-   >>> user = User.objects.get(username="<your_username>")
-   >>> user.roles.append(StaffAreaAdministrator)
-   >>> user.save()
+   > python manage.py create_user <your_username> -s
```

Comment:
I think that's clear from the preceding line that explains the purpose of running the command.

I’ve used the -s switch instead of --staff-area-administrator for ease of typing and memorability. Once familiar
with the short form, developers may not need to refer back to these docs.

Reply:
> IIRC Steve proposed an admin command for adding this role to a user in #4552, but we decided against it.

It's been there since the command was added in #4096 for Airlock purposes. It's definitely easy to overlook -- at one point I started writing one myself until I realized `create_user` can also do updates. I might have called it `create_or_update_user`, though that's a bit of a mouthful. But no less convenient if you have manage.py autocomplete on, and probably less confusing. Hmm.

I support having Create-Update and Delete management commands for models that developers commonly manage locally. More convenient that writing SQL queries, for me at least.